ctpolicy/ctpolicy.go

```diff
@@ -42,14 +44,36 @@ func New(pub pubpb.PublisherClient, sctLogs loglist.List, infoLogs loglist.List,
 	)
 	stats.MustRegister(winnerCounter)
 
+	operatorGroupsGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ct_operator_group_size_gauge",
+			Help: "Gauge for CT operators group size, by operator and log source (capable of providing SCT, informational logs, logs we submit final certs to).",
+		},
+		[]string{"operator", "source"},
+	)
+	stats.MustRegister(operatorGroupsGauge)
+
+	shardExpiryGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ct_shard_expiration_seconds",
+			Help: "CT shard end_exclusive field expressed as Unix epoch time, by operator and logID.",
+		},
+		[]string{"operator", "logID"},
+	)
+	stats.MustRegister(shardExpiryGauge)
+
+	exposeLoglistMetrics(sctLogs, infoLogs, finalLogs, operatorGroupsGauge, shardExpiryGauge)
```

Comment:
Is there a reason why don't just inline all of the logic in this function?

Reply:
Alternatively, keep the helper function, but move the declaration and registration of its two NewGaugeVecs inside the helper as well, so it doesn't need to take them as arguments.

Reply:
Both are good ideas. I'll flip a coin.